### ISSUE TITLE
Floating point exception when running torch.nn.functional.conv_transpose1d

### ISSUE THREAD

**Issue by nimashiri** [pytorch/pytorch#91587]
### 🐛 Describe the bug

When running torch.nn.functional.conv_transpose1d with the following input combination, it throws floating point exception:
```
import torch
import numpy as np
arg_1_tensor = torch.randint(-8192,32,[3, 4], dtype=torch.int64)
arg_1 = arg_1_tensor.clone()
arg_2_tensor = torch.randint(-16384,4096,[3, 3, 3], dtype=torch.int64)
arg_2 = arg_2_tensor.clone()
arg_3_tensor = torch.randint(-1024,256,[3], dtype=torch.int64)
arg_3 = arg_3_tensor.clone()
arg_4_0 = 2
arg_4 = [arg_4_0,]
arg_5 = 2
arg_6_0 = 1
arg_6 = [arg_6_0,]
arg_7 = 0
try:
  res = torch.nn.functional.conv_transpose1d(arg_1,arg_2,arg_3,stride=arg_4,padding=arg_5,output_padding=arg_6,groups=arg_7,)
except Exception as e:
  print("Error:"+str(e))
```

### Versions

PyTorch version: 1.12.0
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 22.04.1 LTS (x86_64)
GCC version: (Ubuntu 11.3.0-1ubuntu1~22.04) 11.3.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.35

Python version: 3.9.15 (main, Nov 24 2022, 14:31:59) [GCC 11.2.0] (64-bit runtime)
Python platform: Linux-5.15.0-56-generic-x86_64-with-glibc2.35
Is CUDA available: True
CUDA runtime version: Could not collect
CUDA_MODULE_LOADING set to:
GPU models and configuration: GPU 0: NVIDIA GeForce GTX 1660 Ti
Nvidia driver version: 525.60.11
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True

Versions of relevant libraries:
[pip3] numpy==1.23.5
[pip3] torch==1.12.0
[pip3] torchaudio==0.12.0
[pip3] torchvision==0.13.0
[conda] blas 1.0 mkl
[conda] cudatoolkit 11.3.1 h2bc3f7f_2
[conda] ffmpeg 4.3 hf484d3e_0 pytorch
[conda] mkl 2021.4.0 h06a4308_640
[conda] mkl-service 2.4.0 py39h7f8727e_0
[conda] mkl_fft 1.3.1 py39hd3c417c_0
[conda] mkl_random 1.2.2 py39h51133e4_0
[conda] numpy 1.23.5 py39h14f4228_0
[conda] numpy-base 1.23.5 py39h31eccc5_0
[conda] pytorch 1.12.0 py3.9_cuda11.3_cudnn8.3.2_0 pytorch
[conda] pytorch-mutex 1.0 cuda pytorch
[conda] torchaudio 0.12.0 py39_cu113 pytorch
[conda] torchvision 0.13.0 py39_cu113 pytorch

cc @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

**Comment by albanD** [comment]
Hi,

Could you try on latest 1.13 please? I get `Error:non-positive groups is not supported` when running this.

**Comment by nimashiri** [comment]
> Hi,
> 
> Could you try on latest 1.13 please? I get `Error:non-positive groups is not supported` when running this.

This bug is fixed in 1.13.1. I got same log message.

**Comment by jbschlosser** [comment]
Closing as addressed in #85248.

**Comment by nimashiri** [comment]
Also this one:

```
import torch
import numpy as np
arg_1_tensor = torch.rand([2, 2, 4], dtype=torch.float64)
arg_1 = arg_1_tensor.clone()
arg_2_tensor = torch.rand([2, 2, 4], dtype=torch.float64)
arg_2 = arg_2_tensor.clone()
arg_3_tensor = torch.rand([4], dtype=torch.float64)
arg_3 = arg_3_tensor.clone()
arg_4_0 = 3
arg_4 = [arg_4_0,]
arg_5_0 = 1
arg_5 = [arg_5_0,]
arg_6_0 = 2
arg_6 = [arg_6_0,]
arg_7 = 0
arg_8_0 = 4
arg_8 = [arg_8_0,]
try:
  res = torch.nn.functional.conv_transpose1d(arg_1,arg_2,arg_3,stride=arg_4,padding=arg_5,output_padding=arg_6,groups=arg_7,dilation=arg_8,)
except Exception as e:
  print("Error:"+str(e))

```
